babel.config.js

```diff
@@ -319,3 +322,49 @@ function pluginToggleBabel8Breaking({ types: t }, { breaking }) {
     },
   };
 }
+
+function pluginPackageJsonMacro({ types: t }) {
+  const fnName = "PACKAGE_JSON";
+
+  return {
+    visitor: {
+      ReferencedIdentifier(path) {
+        if (path.isIdentifier({ name: fnName })) {
+          throw path.buildCodeFrameError(
+            `"${fnName}" is only supported in member expressions.`
+          );
+        }
+      },
+      MemberExpression(path) {
+        if (!path.get("object").isIdentifier({ name: fnName })) return;
+
+        if (path.node.computed) {
+          throw path.buildCodeFrameError(
+            `"${fnName}" does not support computed properties.`
+          );
+        }
+        const field = path.node.property.name;
+
+        // TODO: When dropping old Node.js versions, use require.resolve
```

Comment:
Do we still need to support old versions since 1) we are building on latest node 2) it is a compile-time macro?

Reply:
"compile time" means also "on Node.js 6 inside babel-jest"

Reply:
Oh I thought we are replacing `packageJson.field` on the build step.

Reply:
Yes, but we also run Babel in jest (that's the reason why we cannot use `babel.config.mjs`)